[stable10] Changelog update for upcoming 10.0.8

PVince81 opened the pull request:
38baf9389536fb01f77f96dfff41c36bcdb96b7f..3eb5053ecb6ee2e96e7639e2c6920d991a72598c

Commit:
Changelog update for upcoming 10.0.8

38baf9389536fb01f77f96dfff41c36bcdb96b7f..3eb5053ecb6ee2e96e7639e2c6920d991a72598c

phil-davis commented on CHANGELOG.md:
```diff
@@ -5,40 +5,88 @@ All notable changes to this project will be documented in this file.
+- Added config switch to enable fallback to http scheme when creating fed shares - [#30646](https://github.com/owncloud/core/issues/30646)
+- Added repair step for orphaned sub-shares - [#30695](https://github.com/owncloud/core/issues/30695)
+- Added symfony events for configuration changes (config.php and appconfig) - [#30788](https://github.com/owncloud/core/issues/30788) [#30937](https://github.com/owncloud/core/issues/30937)
+- Added symfony event to let apps resolving private links - [#30911](https://github.com/owncloud/core/issues/30911)
```
s/resolving/resolve

phil-davis commented on CHANGELOG.md:
```diff
@@ -5,40 +5,88 @@ All notable changes to this project will be documented in this file.
+- Validate email address in mail settings section - [#30315](https://github.com/owncloud/core/issues/30315)
+- Only decrypt users who have already logged in with decrypt-all occ command - [#30640](https://github.com/owncloud/core/issues/30640)
+- Replace usage of "create_function" in PHP - [#30714](https://github.com/owncloud/core/issues/30714)
+- Provisioning API can now propertly set default or zero quota - [#30755](https://github.com/owncloud/core/issues/30755)
```
s/propertly/properly

Commit:
More changelog

3eb5053ecb6ee2e96e7639e2c6920d991a72598c..d136a7fad082309b3c309b1a18e880112404e040

PVince81 commented:
Fixed and added a few more entries in af45f5b46050ccefecb6835929a243f61ab167e6 @phil-davis

phil-davis reviewed (approved):
The text looks good. Of course I won't know about items that might be missing ;)
